index.js

```diff
@@ -16,7 +19,9 @@ app.get('/', (req, res) => {
 });
 
 app.get('/chat', (req, res) => {
-  res.render('chat');
+  var randomName = faker.name.findName();
+  console.log(randomName);
+  res.render('chat', {userName: randomName});
```

Comment:
Please follow the style guide. space after curly, etc.

Reply:
ohkk..sorry for that.